### ISSUE TITLE
Constant error pop-ups: 'XtextReconcilerJob' has encountered a problem

### ISSUE THREAD

**Issue by SimonCockx** [eclipse/xsemantics#155]
I'm not sure if this is a bug with XSemantics, XBase or XText, but it is an annoying one. @LorenzoBettini maybe you know? If this is not an XSemantics issue, I can post the issue elsewhere.

When working on more complex rules, I constantly get an error message pop-up like in the image below. I can just click it away and continue without problems, but it is really annoying to have that pop-up every time I make a small change. I've managed to create a minimal example that reproduces the problem.

Steps to reproduce:
1. Create a default XText project.
2. Add the runtime of XSemantics to its dependencies.
3. Create a file `MyDsl.xsemantics` and add the following code.
```
system org.xtext.example.mydsl.typing.MyDslTyping

import org.xtext.example.mydsl.myDsl.Greeting

judgments {
	inferName |- Greeting g : output String
}

rule InferName
	G |- Greeting g : String n
from {
	if (true) {
		n = [REPLACE ME]
	}
}
```
4. Replace `[REPLACE ME]` with `g.name`, then replace it with `g.test`. Every time you switch between `g.name` and `g.test`, the error pops up.

![reconcilerjob](https://user-images.githubusercontent.com/47859223/157022548-45f530a7-0ca5-450e-a3cd-1c6579579acc.PNG)

**Comment by cdietrich** [comment]
https://github.com/eclipse/xtext-xtend/issues/1067

**Comment by LorenzoBettini** [comment]
I found a fix for my other projects. I forgot to apply that also on Xsemantics. I'll try to do that ASAP and release a fixed version.

**Comment by lwrage** [comment]
Is there a chance you could apply the fix in the next Xsemantics release?

**Comment by LorenzoBettini** [comment]
@lwrage OK, but then I'll probably delay the release to the beginning of the week :)

**Comment by LorenzoBettini** [comment]
Closes by #167 

Will be part of the new release hopefully today
